src/components/select-field/SelectField.scss

```diff
@@ -54,7 +55,9 @@
 .bdl-SelectFieldDropdown {
     z-index: 4;
     min-width: 100%;
+    margin-top: $bdl-grid-unit;
```

Comment:
Do we have any use cases where a flush menu is still the desired experience?

Reply:
No, we don't